crates/sequencer/src/commitment_controller.rs

```diff
@@ -26,6 +26,7 @@ pub fn get_commitment_info(
     ledger_db: &LedgerDB,
     min_soft_confirmations_per_commitment: u64,
     prev_l1_height: u64,
+    state_diff_threshold_reached: bool,
```

Comment:
Note to reviewers: This is more like a `force` flag to skip check for `min_soft_confirmations_per_commitment`